front/js/service.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Create a scratchy service.
+ * @constructor
+ * @param {string} apiUrl - Scratchy service API URL, eg: "http://localhost:5000"
+ * @param {string} roomID - room id , eg:60895dd62d1a706830c31f10
+ */
+function ScratchyService(apiUrl) {
+    let _this = this;
+    this.cacheUsers = []; // List of users, used as cache
+
+    // ROOM 
+
+    this.fetchRooms = function(){
+        console.debug("ScratchyService.fetchRooms()");
+        console.info(_this.cacheUsers);
+        return [];
+    };
+//                               ROOM FUNCTIONS
+
+    this.updateRoom = async function(roomId,roomTitle,roomDescription){
+        const reponse = await axios.put(apiUrl+'/room/'+roomId, { title: roomTitle , description: roomDescription });
+        return reponse.data;
+    };
+
+    this.createRoom = async function(roomTitle,roomDescription){
+        const reponse = await axios.post(apiUrl+'/room', { title: roomTitle , description: roomDescription });
+        return reponse.data;
+    };
+
+    this.deleteRoom = async function(roomId){
+
+        const reponse = await axios.delete('http://localhost:5000/api/room/'+roomId);
+        return reponse.data;
+    };
+
+    this.getAllRooms = async function(){ // put the id between "" 
```

Comment:
Why do you have this comment `// put the id between "" ` ? They are no "id" parameter